### ISSUE TITLE
[Request]Barries setting and items/pokemons management

### ISSUE THREAD

**Issue by kr1dog** [NECROBOTIO/NecroBot#4657]
It would be great if we could change the number of usable berries and / or set the use of berries only for certain pokemon to put as input in a list in config.json.

It would also be very useful an interface to allow the management of items and pokemons in real time to the user, or at least someway to do this without an interface.

Anyway I want to congratulate you for your great work and forgive me if this post does not respect some rules that I have not read

**Comment by nfettinger** [comment]
You can use your configuration to set this up in a general sense. Limiting berry use to certain pokemon seems like too constrained of a use case to me.

**Comment by kr1dog** [comment]
Yes, but when one has to farm certain pokemons this feature becomes very useful for me as sniping consumes a lot of berries and balls, therefore one would prefer to use the berries for certain pokemon rather than for other, but this of course is just my opinion.
